todo_app/data/Item.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+class Item:
+    def __init__(self, id, name, desc, idList, due_date, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.description = desc
+        self.idList = idList
+        self.dueDate = due_date
+        self.status = status        
+
+    @classmethod
+    def from_trello_card(cls, card):
+        completed_list_id = os.getenv('CORNDEL_TODO_LIST_ID')
+        trello_idList = card['idList']
+        if(trello_idList == completed_list_id):
+            return cls(card['id'], card['name'], card['desc'], trello_idList, card['due'])
+        else:
+            return cls(card['id'], card['name'], card['desc'], trello_idList, card['due'], 'Done')
```

Comment:
More as an alternative to see rather than a suggestion, there's a fair amount duplicated here but I'm not sure if the alternative is neater - do you like the below any better?
```suggestion
        trello_idList = card['idList']
        status = "To Do" if trello_idList == completed_list_id else "Done"
        return cls(card['id'], card['name'], card['desc'], trello_idList, card['due'], status)
```

Reply:
Thanks.. Ive incorporated all of the changes.  Agreed the suggestion on setting a status variable before returning is a cleaner approach